tests/test_image.py

```diff
@@ -53,10 +53,13 @@ def test_ImageWriter(tmp_path):
     stream = pipeline.transform_stream()
     pipeline.run()
 
-def test_ImageReader():
+@pytest.mark.xfail(strict=True)
+def test_ImageReader(data_path):
+    d = data_path / "images"
+    d.mkdir()
+    p = d / "test_image_3.png"
```

Comment:
This fails when I give it the existing path saying 

> Cannot create a file when that file already exists
and when I give some other new path it fails again and says 
> No such file or directory